fedimint-client/src/api_announcements.rs

```diff
@@ -44,11 +44,21 @@ pub async fn run_api_announcement_sync(client_inner: Arc<Client>) {
         let results = join_all(client_inner.api.all_peers().iter()
             .map(|peer_id| async {
                 let peer_id = *peer_id;
-                let announcements = client_inner
-                    .api
-                    .api_announcements(peer_id)
-                    .await
-                    .with_context(move || format!("Fetching API announcements from peer {peer_id} failed"))?;
+                let announcements =
+
+                retry(
+                    "Fetch api announcement (sync)",
+                    backoff_util::aggressive_backoff(),
```

Comment:
This will only retry for ~20s. Should use a strategy that retries longer?

Reply:
I think 20s is plenty comparing to one time. :)